tests/cypress/integration/5-AI-SiteGen-onboarding-flow/fork.cy.js

```diff
@@ -0,0 +1,81 @@
+// <reference types="Cypress" />
+
+import { AdminBarCheck, DarkBGCheck, LightBGChcek } from "../wp-module-support/siteGenBG.cy";
+
+describe( 'SiteGen Fork Step', function () {
+	before( () => {
+		cy.visit(
+			'wp-admin/?page=nfd-onboarding#/wp-setup/step/fork'
+		);
+	} );
+
+    it( 'Check for the header admin bar', () => {
+        AdminBarCheck();
+    } );
+
+    it( 'Check for the existing dark background', () => {
+		DarkBGCheck();
+	} );
+
+    it( 'Check for the light background', () => {
+        LightBGChcek();
+    } );
+
+    it( 'Check for the heading and the title', () => {
+        cy.get( '.nfd-onboarding-step__heading__title' )
+            .should('be.visible')
+            .should('have.text', 'Welcome to WordPress');
+    } );
+
+    it ( 'Check for the subheading', () => {
+        cy.get( '.nfd-onboarding-step__heading__subtitle' ).should('be.visible');
+    } );
+
+    const OptionsDetails = (className,textValue,optionsValue) => {
```

Comment:
Please move this to the variable declaration section. Lets not keep it between the tests.

Reply:
This is done.